components/equihash/src/tromp.rs

```diff
@@ -30,23 +31,30 @@ extern "C" {
     fn equi_digiteven(eq: *mut CEqui, r: u32, id: u32);
     fn equi_digitK(eq: *mut CEqui, id: u32);
```

Comment:
Since we're not using threads, we could remove `id` from all these functions and replace it with zero in C.